index.html

```diff
@@ -55,10 +60,13 @@ <h3 class="connect">LET’S CONNECT</h3>
             <section class="projects">
          
                 <div class="project-container">
-
+                <section class="project-image">
                     <div class="section1">
-    
+                        <!-- <img src="images/Snapshoot-Portfolio-Disk1.png" alt=""> -->
                     </div>
+                </section>
+                
+                <section class="project-description">
                     <div>
                         <h2>Tonic</h2>
                         <ul class="tonic-title">
```

Comment:
- [ ] Kindly follow the Figma design as much as possible, this includes `font-size`, text `color`, and the `background-color` of the technologies badges 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/194669242-10c93a90-e873-499f-a52b-aaec531061e7.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/194669263-9e587e97-915c-44f0-8eab-b4a57f466ba3.png)